docs/data_requirements/input_data.md

```diff
@@ -13,7 +13,9 @@ The **oedatamodel-parameter** ([docs](https://github.com/sedos-project/oedatamod
 
    
 
-## Data tables and table naming conventions
+## Data tables 
+
+### Table naming conventions
```

Comment:
My suggestion would be to move this below the next section and simply call it "table naming".